.env.example

```
@@ -12,8 +12,19 @@ EHRBASE_PASSWORD=
 # CORS origins - for API service (JSON array)
 CORS_ORIGINS=["http://localhost:5173"]
 
+# OIDC - for API service
+OIDC_ISSUER=http://localhost:5556/dex
+OIDC_CLIENT_ID=open-cis-web
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OIDC_CLIENT_ID key should go before the OIDC_ISSUER key

(UnorderedKey)